src/commands/resetlead/ResetUtil.ts

```diff
@@ -0,0 +1,63 @@
+import moment, { Moment } from "moment-timezone";
+
+export enum WvwRegion {
+    EU = "EU",
+    NA = "NA"
+}
+
+function getWvwRegionProperties(wvwRegion: WvwRegion) {
+    let resetWeekDay: number;
+    let resetTimeUTC: number;
+    switch (wvwRegion) {
+        case WvwRegion.EU:
+            resetWeekDay = 5;
+            resetTimeUTC = 18;
+            break
+        case WvwRegion.NA:
+            resetWeekDay = 6
+            resetTimeUTC = 2
+            break
+        default:
+            throw Error("Unmapped WvW Region")
+    }
+    return {resetWeekDay, resetTimeUTC};
```

Comment:
I do not know this syntax and trying it out in repl.it produces a syntax error. Did you mean to use the tuple (or in JS: array) composition syntax

```javascript
return [resetWeekDay, resetTimeUTC}];
```

with brackets instead of braces?

Or are you emplying the record syntax, which seems to be preceeded by a diamond? https://github.com/tc39/proposal-record-tuple

Reply:
Those are anonymous objects. 
Typescript is inferring the attribute names and types.
![image](https://user-images.githubusercontent.com/5864045/108530319-c3973800-72d5-11eb-8a92-29da4c9573a5.png)

Reply:
Ooh, interesting shorthand! The more you know.